src/coreclr/nativeaot/Runtime/wasm/PalCreateDump.cpp

```diff
@@ -0,0 +1,53 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+// We do not support crash dumps in Wasm yet, so just stub.
+
+#include "../unix/PalCreateDump.h"
```

Comment:
```suggestion
#include "PalCreateDump.h"
```
I think that should work...

Reply:
Yes, it does, thanks.